generator/src/main/java/com/scottlogic/deg/generator/validators/GenerationConfigValidator.java

```diff
@@ -19,6 +32,31 @@ public ValidationResult validateCommandLine(GenerationConfig config) {
             errorMessages.add("RANDOM mode requires max row limit: use -n=<row limit> option");
         }
 
+        if (configSource.shouldViolate()) {
+            checkViolationGenerateOutputTarget(errorMessages, outputTarget);
+        } else {
+            checkGenerateOutputTarget(errorMessages, outputTarget);
+        }
+
         return validationResult;
     }
-}
\ No newline at end of file
+
+    private void checkGenerateOutputTarget(ArrayList<String> errorMessages, OutputTarget outputTarget) {
+        if (outputTarget.isDirectory()) {
+            errorMessages.add("Invalid Output - target is a directory, please use a different output filename");
```

Comment:
I think this belongs in an OutputTargetValidator, specifically a FileOutputTargetValidator

Reply:
Shall I update the issue acceptance criteria before changing the code?